spyderlib/config/main.py

```diff
@@ -719,7 +719,7 @@ def is_gtk_desktop():
               'zenburn/background':  "#3f3f3f",
               'zenburn/currentline': "#333333",
               'zenburn/currentcell': "#2c2c2c",
-              'zenburn/occurence':   "#7a738f",
+              'zenburn/occurrence':   "#7a738f",

```

Comment:
Changing option names requires a *major* bump in `CONF_VERSION` number (present below in this same file)